Code/PgSQL/rdkit/mol_op.c

```diff
@@ -376,7 +378,7 @@ fmcs_mol2s_transition(PG_FUNCTION_ARGS) {
     }
     if(PG_ARGISNULL(0) && !PG_ARGISNULL(1)) { // first call
 ///elog(WARNING, "fmcs_mol2s_transition() called first time");
-        CROMol mol = PG_GETARG_DATUM(1);
+        CROMol mol = (CROMol)PG_GETARG_DATUM(1);
```

Comment:
We've been trying to move away from C-style casts in the code. Please use `static_cast` For these.